app/frontend/src/features/communities/events/EventUsers.tsx

```diff
@@ -0,0 +1,80 @@
+import { Card, CircularProgress, Typography } from "@material-ui/core";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import UserSummary from "components/UserSummary";
+import useUsers from "features/userQueries/useUsers";
+import { Error as GrpcError } from "grpc-web";
+import makeStyles from "utils/makeStyles";
+
+import { SEE_ALL } from "./constants";
+
+const useStyles = makeStyles((theme) => ({
+  cardSection: {
+    padding: theme.spacing(2),
+  },
+  organisers: {
+    display: "grid",
+    marginBlockStart: theme.spacing(2),
+    rowGap: theme.spacing(1),
```

Comment:
It doesn't matter, but why use grid here?

Reply:
Less code to write for creating spacing between users (`rowGap` as opposed to the `& > * + *: { marginTop: theme.spacing(1) }`). I think I'm slowly being converted to the grid way of doing things in these cases 😂